stdlib/date/time_fns.flux

```diff
@@ -0,0 +1,60 @@
+package date
```

Comment:
Can you rename this file to `date.flux` to be consistent with the other packages?